src/Routes.js

```diff
@@ -14,21 +14,21 @@ import MainYeseul from './pages/Yeseul/Main/Main';
 
 class Routes extends React.Component {
   render() {
-    return(
+    return (
       <Router>
         <Switch>
           <Route exact path="/login-doeun" component={LoginDoeun} />
           <Route exact path="/main-doeun" component={MainDoeun} />
-          <Route exact path="/login-yesl" component={LoginHwisung} />
-          <Route exact path="/main-yesl" component={Mainhwisung} />
-          <Route exact path="/login-hwisung" component={LoginYeonju} />
-          <Route exact path="/main-hwisung" component={MainYeonju} />
-          <Route exact path="/login-yeonju" component={LoginYeseul} />
-          <Route exact path="/main-yeonju" component={MainYeseul} />
+          <Route exact path="/" component={LoginHwisung} />
+          <Route exact path="/Main" component={MainHwisung} />
```

Comment:
휘성님 ㅋㅋㅋㅋ 휘성님이 첫페이지 차지하셨네요 😂 
휘성님 페이지 경로도 네이밍 맞춰주셔야 컨플릭트 안나실고에오 (순서수정은 덤 ㅎㅎㅎ)
```suggestion
          <Route exact path="/login-hwisung" component={LoginHwisung} />
          <Route exact path="/main-hwisung" component={MainHwisung} />
```